test/unit-test/test_keystore.c

```diff
@@ -209,6 +209,17 @@ static void _test_keystore_get_xpub(void** state)
     wally_free_string(xpub_string);
 }
 
+static void _test_keystore_get_root_fingerprint(void** state)
+{
+    mock_state(_mock_seed, _mock_bip39_seed);
+    char fingerprint[4];
+    assert_true(get_root_fingerprint(fingerprint, sizeof(fingerprint)));
+    assert_int_equal(fingerprint[0], 0xffffffffffffffca);
```

Comment:
`assert_memory_equal` easier to compare the whole fingerprint at once? it is weird with the 0xfff... padding / int conversion.